Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -186,6 +187,33 @@ void ProjectData::buildProcesses()
                     *_mesh_vec[0], *nl_slv, std::move(time_disc),
                     std::move(conv_crit), _process_variables, _parameters, pc));
         }
+        else if (type == "SMALL_DEFORMATION")
+        {
+            switch (pc.getConfigParameter<int>("dimension"))
```

Comment:
I think, under the mentioned cases, we only need dimension of element and the dimension of space (plates, shells, etc,), which are already kept in mesh object.